tools/wptrunner/wptrunner/testrunner.py

```diff
@@ -723,9 +723,12 @@ def after_test_end(self, test, restart, force_rerun=False, force_stop=False):
             test, test_group, group_metadata = self.get_next_test()
             if test is None:
                 return RunnerManagerState.stop(force_stop)
-            restart = (self.restart_on_new_group and
-                       test_group is not self.state.test_group)
-            if restart:
+            if (
```

Comment:
Can we change the argument name to e.g. `force_restart` and then write the code as

```
restart = force_restart or (self.restart_on_new_group and test_group is not self.state.test_group)
```
it could even go outside the if/else clause.

Reply:
I made a change in https://github.com/web-platform-tests/wpt/pull/34483, per this suggestion.

Feel free to continue with either one. And sorry for the breakage...

Reply:
Looks like we can not move the line James proposed out of the if/else clause because test_group will only be able at line 723. Due to this, rename restart to force_restart won't bring us any benefit.

Due to this, I suggest we go ahead with this fix.